python-frontend/typeshed_serializer/tests/conftest.py

```diff
@@ -31,6 +31,8 @@
 
 @pytest.fixture(scope="session")
 def typeshed_stdlib():
+    typeshed_serializer.STDLIB_PATH = "resources/mock_stdlib"
+    typeshed_serializer.CURRENT_PATH = os.path.dirname(__file__)
```

Comment:
Here can we use the CURRENT PATH constant instead? Genuine question not sure when the fixture is invoked.

Reply:
Thanks, good catch! I somehow missed it.